app/src/main/kotlin/cz/covid19cz/erouska/ui/help/HelpVM.kt

```diff
@@ -1,10 +1,29 @@
 package cz.covid19cz.erouska.ui.help
 
 import androidx.hilt.lifecycle.ViewModelInject
+import androidx.lifecycle.viewModelScope
+import arch.livedata.SafeMutableLiveData
+import cz.covid19cz.erouska.AppConfig
 import cz.covid19cz.erouska.ui.base.BaseVM
 import cz.covid19cz.erouska.ui.help.event.HelpCommandEvent
+import cz.covid19cz.erouska.utils.L
+import cz.covid19cz.erouska.utils.Markdown
+import kotlinx.coroutines.CancellationException
+import kotlinx.coroutines.Job
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.launch
+import java.util.regex.Pattern
 
-class HelpVM @ViewModelInject constructor() : BaseVM() {
+class HelpVM @ViewModelInject constructor(
```

Comment:
You should not reformat this. Do you use some custom code formatting? Use the one we have included in the project.